domains/domains/cravetea.json

```
@@ -0,0 +1 @@
+
```

```suggestion
{
  "owner": {
    "username": "arbashscrpt",
    "email": "mias42855@gmail.com"
  },
  "records": {
    "CNAME": "crave-tea-tunnel.trycloudflare.com"
  }
}
```

invalid Domain Structure please read -> https://docs.is-a.dev/domain-structure/